bin/test/main.ml

```diff
@@ -60,6 +60,19 @@ let ansi_color_strip str =
   in
   loop 0
 
+let trim_end s =
+  let l = String.length s in
+  let rec f i =
+    if i > l
+    then
+      i
+    else
+      match s.[l - i - 1] with
+      | ' ' | '\t' | '\011'..'\013' -> f (succ i)
+      | _ -> i
+  in
+  String.sub s 0 (l - (f 0))
```

Comment:
This project depends on Astring so you can implement this like:
```ocaml
let trim_end s =
  String.drop ~rev:true ~sat:(function ' ' | '\t' | '\011'..'\013' -> true | _ -> false) s
```

Reply:
You can probably go even further and make it:
```ocaml
let trim_end s = String.drop ~rev:true ~sat:Ascii.Char.is_white s
```

Or Ascii.Char.is_blank, not sure which one suits this better!